packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -199,15 +179,7 @@ export default {
         createGuestUser: async ({ commit }, {
             url, tenant, data, timeout, otacToAuthExchanger
         }) => {
-            const config = {
-                headers: {
-                    'Content-Type': 'application/json',
-                    'Accept-Tenant': tenant
-                },
-                timeout
-            };
-
-            const response = await axios.post(url, data, config);
+            const response = await accountApi.createGuestUser(url, data, timeout, tenant);
             // eslint-disable-next-line no-unused-vars
             const otac = response.data.token;
```

Comment:
Is the HTTP errors then `response.data` will be `undefined` and will error here.

Reply:
@LTurns, you can use the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here.